webpack/scenes/ContentViews/Details/Versions/ContentViewVersionErrata.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  BugIcon,
+  SecurityIcon,
+  EnhancementIcon,
+} from '@patternfly/react-icons';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import './ContentViewVersionErrata.scss';
+
+const ContentViewVersionErrata = ({ cvId, versionId, errataCounts }) => {
+  const {
+    total,
+  } = errataCounts;
+
+  const errataIcons = {
+    security: SecurityIcon,
+    bugfix: BugIcon,
+    enhancement: EnhancementIcon,
+  };
+
+  const getHref = type => `/content_views/${cvId}/versions/${versionId}/errata?queryPagedSearch=%20type%20%3D%20${type}`;
+
+  const ErrataLinkwithIcon = () => Object.keys(errataIcons).map((type) => {
+    const ErrataIcon = errataIcons[type];
+    return (
+      <React.Fragment key={type}>
+        <ErrataIcon title={type} />
+        <a className="errata-icons-with-commas" href={getHref(type)}> {`${errataCounts[type] || '0'}`} </a>
+      </React.Fragment>
+    );
+  });
+
+  return (
+    <React.Fragment>
+      <a href={urlBuilder(`content_views/${cvId}/versions/${versionId}/errata`, '')}>{`${total || 0} `}</a>
+      ( <ErrataLinkwithIcon /> )
```

Comment:
This results in "( " and " )" (with spaces) and makes the right parenthesis look oddly spaced:
![parens](https://user-images.githubusercontent.com/22042343/116466013-f9a7e980-a83b-11eb-94a0-6a93e196d352.png)

Reply:
Move some CSS around to remove the margin-right from the last child.
![Screenshot from 2021-04-28 17-01-05](https://user-images.githubusercontent.com/21146741/116472030-58bd2c80-a843-11eb-8652-f3f5e48abd51.png)